lightpath/path.py

```diff
@@ -67,46 +80,50 @@ class DeviceState(enum.Enum):
     Error = 5
 
 
-def find_device_state(device):
+def find_device_state(device: Device) -> Tuple[DeviceState, LightpathState]:
     """
     Report the state of a device
 
     The device must implement ``.inserted`` and ``removed``.
```

Comment:
Stale docstring, maybe?

Reply:
I've gone blind looking at this code I guess